VMAccess/vmaccess.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
The final commit should keep this as python not python3.  We don't expect distros that have moved completely to Python3 to keep the binary as python3 and/or they will introduce a symlink for python that refers to python3.

Reply:
Sure. I'll revert this once the Tests work.